affiliated/registry.json

```
@@ -716,7 +716,7 @@
         },
         {
             "name": "Halotools",
-            "maintainer": "Andrew Hearin <andrew.hearin@yale.edu>",
+            "maintainer": "Andrew Hearin <ahearin@nal.gov>, Nick van Alfen",
```

I think the email has to be in the end. As it stands, Nick's name is missing from preview.

```suggestion
            "maintainer": "Andrew Hearin, Nick van Alfen <ahearin@nal.gov>",
```

---

Dang, sorry, should now be fixed.